.github/workflows/generator-publish-sources.yaml

```diff
@@ -0,0 +1,45 @@
+name: Generate and Publish Sources
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: 'SDK Name'
+        required: true
+        type: string
+      version:
+        description: 'SDK Version'
+        required: true
+        type: string
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: sdk
+          path: openworld/sdk/generator/package
+      - uses: actions/download-artifact@v3
+        with:
+          name: specs
+          path: openworld/sdk/generator/package
+      - name: Publish Code
+        working-directory: openworld/sdk/generator/package
+        run: |
+          git config --global user.email "oss@expediagroup.com"
+          git config --global user.name "Expedia Group Open Source"
+          rm -rf ../../../../release/"${{github.event.inputs.name}}"
+          mkdir -p ../../../../release/"${{github.event.inputs.name}}"
+          tar -xzf *.tar.gz
+          rm *.tar.gz
+          find ./ -name \*.py -exec cp {} ../../../../release/"${{github.event.inputs.name}}"/ \;
```

Comment:
Will this exclude the middle folder named `openworld-sdk-python-fraudpreventionv2-1.1.0`?

Reply:
Yes.